flow/initialize/init.go

```diff
@@ -22,76 +22,68 @@ import (
 	"fmt"
 	"log"
 
+	"github.com/onflow/flow-cli/flow/cli"
 	"github.com/onflow/flow-go-sdk/crypto"
 	"github.com/psiemens/sconfig"
 	"github.com/spf13/cobra"
-
-	cli "github.com/onflow/flow-cli/flow"
 )
 
-type Config struct {
+type Flags struct {
 	ServicePrivateKey  string `flag:"service-priv-key" info:"Service account private key"`
 	ServiceKeySigAlgo  string `default:"ECDSA_P256" flag:"service-sig-algo" info:"Service account key signature algorithm"`
 	ServiceKeyHashAlgo string `default:"SHA3_256" flag:"service-hash-algo" info:"Service account key hash algorithm"`
 	Reset              bool   `default:"false" flag:"reset" info:"Reset flow.json config file"`
 }
 
 var (
-	conf Config
+	flags Flags
 )
 
 var Cmd = &cobra.Command{
 	Use:   "init",
 	Short: "Initialize a new account profile",
 	Run: func(cmd *cobra.Command, args []string) {
-		if !cli.ConfigExists() || conf.Reset {
-			var pconf *cli.Config
-			serviceKeySigAlgo := crypto.StringToSignatureAlgorithm(conf.ServiceKeySigAlgo)
-			serviceKeyHashAlgo := crypto.StringToHashAlgorithm(conf.ServiceKeyHashAlgo)
-			if len(conf.ServicePrivateKey) > 0 {
-				serviceKey := cli.MustDecodePrivateKeyHex(serviceKeySigAlgo, conf.ServicePrivateKey)
-				pconf = InitProjectWithServiceKey(serviceKey, serviceKeyHashAlgo)
-			} else {
-				pconf = InitProject(serviceKeySigAlgo, serviceKeyHashAlgo)
+
```

Comment:
I wonder if we should just go ahead and remove this command entirely and replace with `flow project init`.

This implementation is outdated and tightly coupled to the emulator, which I don't like.